The patch that would have made the NSLU2
kernel non compatible with other ixp4xx machs
never entered the kernel. So it is actually
safe to remove the prompt dependencies.

Signed-off-by: Alessandro Zummo <a.zummo@towertech.it>
---
 arch/arm/mach-ixp4xx/Kconfig |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

--- linux-ixp4xx.orig/arch/arm/mach-ixp4xx/Kconfig	2006-03-08 10:08:05.000000000 +0100
+++ linux-ixp4xx/arch/arm/mach-ixp4xx/Kconfig	2006-03-08 10:10:23.000000000 +0100
@@ -8,11 +8,9 @@ menu "Intel IXP4xx Implementation Option
 
 comment "IXP4xx Platforms"
 
-# This entry is placed on top because otherwise it would have
-# been shown as a submenu.
 config MACH_NSLU2
 	bool
-	prompt "NSLU2" if !(MACH_IXDP465 || MACH_IXDPG425 || ARCH_IXDP425 || ARCH_ADI_COYOTE || ARCH_AVILA || ARCH_IXCDP1100 || ARCH_PRPMC1100 || MACH_GTWX5715)
+	prompt "Linksys NSLU2"
 	help
 	  Say 'Y' here if you want your kernel to support Linksys's
 	  NSLU2 NAS device. For more information on this platform,
